### PR TITLE
Bulk import from homebrew/versions (2/7)

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Import formulae from homebrew/versions. Those that fail to build will likely be killed.

### CODE REVIEW

File: Formula/ffmpeg@2.8.rb [hunk 1]
```diff
@@ -0,0 +1,190 @@
+class FfmpegAT28 < Formula
+  desc "Play, record, convert, and stream audio and video"
+  homepage "https://ffmpeg.org/"
+  url "https://ffmpeg.org/releases/ffmpeg-2.8.6.tar.bz2"
```
Comment (retokromer) [inline Formula/ffmpeg@2.8.rb]: The last release of the 2.8 branch is 2.8.11 not the 2.8.6 (the current last release is 3.2.4).

**Comment by ilovezfs** [comment]
CC @woodruffw RE: dylib ID of libglfw.dylib

**Comment by ilovezfs** [comment]
@MikeMcQuaid 🍏 modulo the unholy download.sourceforge.net false positives.

**Comment by MikeMcQuaid** [comment]
Thanks @ilovezfs ❤️

**Pull request closed without merging**